### PR TITLE
feat(ui): Peer status indicator

### PR DESCRIPTION
**Pull request by juliendonck**
Indicator to that shows if you are in sync with the maintainer's version and instructions on how to pull that version into your main branch:

Screenshot 2021-05-11 at 09 53 05

Signed-off-by: Julien Donck <hello@juliendonck.com>

### CODE REVIEW

File: ui/DesignSystem/Component/SourceBrowser/FileView/PeerStatus.svelte [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<script>
+  import { Avatar, Icon } from "../../../Primitive";
```
Comment (geigerzaehler) [inline ui/DesignSystem/Component/SourceBrowser/FileView/PeerStatus.svelte:2]: We can use absolute imports now.
```suggestion
  import { Avatar, Icon } from "ui/DesignSystem/Primitive";
```

**Comment by geigerzaehler** [comment]
We don’t have an API yet that provides this information. I have  two questions to make sure I understand this correctly.
* Do we show this for every branch? I.e. branches created by the user and branches created by remote peers?
* Do we always compare against the default branch of the maintainer?
I’ll create an issue to implement the API.

**Comment by juliendonck** [comment]
> We don’t have an API yet that provides this information. I have two questions to make sure I understand this correctly.
> 
>     * Do we show this for every branch? I.e. branches created by the user and branches created by remote peers?

I think eventually I'd like to show a dropdown in the overlay where you can select any peer and it shows you the difference. For now I think just showing the maintainer main branch would be enough.

> 
>     * Do we always compare against the default branch of the maintainer?

I think initially it should just be to merge from main -> main. In the end the instructions should teach you how to merge other ones in if you want to.

> 
> 
> I’ll create an issue to implement the API.

**Comment by geigerzaehler** [comment]
It looks like the API described in https://github.com/radicle-dev/radicle-surf/issues/174 should be enough to implement this. It looks to me like the change in `radicle-surf` is a bit more involved because the APIs don’t quite fit this use case but it  should be doable.

**Comment by juliendonck** [comment]
Closed for now until we have bandwidth to work on this.

**Pull request closed without merging**